agent/internal/testutil/task_config.go

```diff
@@ -17,7 +19,16 @@ func MakeTaskConfigFromModelData(ctx context.Context, settings *evergreen.Settin
 	if err != nil {
 		return nil, errors.Wrap(err, "getting global GitHub OAuth token")
 	}
-	exp, err := model.PopulateExpansions(data.Task, data.Host, oauthToken)
+	appToken, err := settings.CreateInstallationToken(ctx, data.Project.Owner, data.Project.Repo, nil)
+	if err != nil {
+		grip.Debug(message.WrapError(err, message.Fields{
```

Comment:
this function is only used in tests if that changes anything (might be helpful to include that in the comment above)